### PR TITLE
add how-to-create-a-subproject.md for sig-data-science

### PR DESCRIPTION
**Pull request by MichaelClifford**
I created a draft doc with step by step instructions for how to create a new subproject within sig-data-science. It was created while working on #25 and creating our sig's first subproject

Let's all use, review and updated this document as we see fit as we start our first subprojects. Any and all feedback very welcome :)

### CODE REVIEW

File: sig-data-science/how-to-create-a-subproject.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+ - description: Example Subproject
+    day: Day of Week
+    time: "Time"
+    tz: ET (East Time)
+    frequency: weekly
+    url: google meet link
+```
+
+Under `sigs` → `dir:sig-data-scienc`e find the `subprojects` section and add the new subproject name, description and link to the original repo's OWNERS file.
```
Comment (aakankshaduggal) [inline sig-data-science/how-to-create-a-subproject.md:61]:
```suggestion
Under `sigs` → `dir:sig-data-science` find the `subprojects` section and add the new subproject name, description and link to the original repo's OWNERS file.
```

File: sig-data-science/how-to-create-a-subproject.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+# Steps to create a new subproject under SIG-Data-Science:
+
+
+## Step 1:
+Make sure you have met all the prerequisite needed for a subproject:
```
Comment (aakankshaduggal) [inline sig-data-science/how-to-create-a-subproject.md:5]:
```suggestion
Make sure you have met all the prerequisites needed for a subproject:
```

File: sig-data-science/how-to-create-a-subproject.md (hunk 1 above)
Comment (oindrillac) [inline sig-data-science/how-to-create-a-subproject.md:61]:
```suggestion
Under `sigs` → `dir:sig-data-science` find the `subprojects` section and add the new subproject name, description and link to the original repo's OWNERS file.
```

File: sig-data-science/how-to-create-a-subproject.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    description: '[Described below](#example-subproject)'
+    owners:
+    - https://raw.githubusercontent.com/<subproject-repo>/OWNERS
+
+
+```
+
+## Step 3:
+
+Push your changes to GitHub and open a PR in the upstream repo https://github.com/open-services-group/community.
```
Comment (oindrillac) [inline sig-data-science/how-to-create-a-subproject.md:77]:
```suggestion
Push your changes to your GitHub fork and open a PR in the upstream repo https://github.com/open-services-group/community.
```

**Review by oindrillac — COMMENTED** [review]
thanks for putting this together @MichaelClifford. Some small nits

File: sig-data-science/how-to-create-a-subproject.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+* `sigs.yaml`
+
+### `sig-data-science/README.md`
+
+In `sig-data-science/README.md` make an update to the “Details about SIG-Data-Science Subprojetcs” section with the title and a short description of the subproject.
+
+```markdown
+<!-- BEGIN CUSTOM CONTENT -->
+
+`sig-data-science/README.md`# Details about SIG-Data-Science Subprojects
```
[the remark this answers is not in the document]
Comment (MichaelClifford) [reply sig-data-science/how-to-create-a-subproject.md:27]: :+1:  this is a mistake

File: sig-data-science/how-to-create-a-subproject.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+## Step 2:
+
+Create a feature branch on your fork of `open-services-group/community` and make changes to the following two files:
+* `sig-data-science/README.md`
+* `sigs.yaml`
+
+### `sig-data-science/README.md`
+
+In `sig-data-science/README.md` make an update to the “Details about SIG-Data-Science Subprojetcs” section with the title and a short description of the subproject.
```
Comment (suppathak) [inline sig-data-science/how-to-create-a-subproject.md:22]:
```suggestion
In `sig-data-science/README.md` make an update to the “Details about SIG-Data-Science Subprojects” section with the title and a short description of the subproject.
```

File: sig-data-science/how-to-create-a-subproject.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+```markdown
+<!-- BEGIN CUSTOM CONTENT -->
```
[the remark this answers is not in the document]
Comment (MichaelClifford) [reply sig-data-science/how-to-create-a-subproject.md:25]: The readme is auto generated, so any custom content we want to add independent of the auto-generation process has to be within the CUSTOM CONTENT tags.

File: sig-data-science/how-to-create-a-subproject.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+`sig-data-science/README.md`# Details about SIG-Data-Science Subprojects
+
+## Example Subproject 1
+
+Description…
+
+## Example Subproject 2
+
+Description
```
Comment (suppathak) [inline sig-data-science/how-to-create-a-subproject.md:35]:
```suggestion
Description...
```

File: sig-data-science/how-to-create-a-subproject.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (oindrillac) [reply sig-data-science/how-to-create-a-subproject.md:25]: 👍

**Comment by MichaelClifford** [comment]
Thanks @suppathak @oindrillac @aakankshaduggal updates made :)

**Review by oindrillac — COMMENTED** [review]
/lgtm

**Comment by MichaelClifford** [comment]
cc @open-services-group/steering

**Review by aakankshaduggal — COMMENTED** [review]
/lgtm
Thanks.

**Review by erikerlandson — APPROVED** [review]

**Pull request merged**